src/vs/workbench/contrib/inlineChat/browser/inlineChatDecorations.ts

```diff
@@ -72,6 +80,10 @@ export class InlineChatDecorationsContribution extends Disposable implements IEd
 		this._onEnablementOrModelChanged();
```

Comment:
I would put `this._updateCurrentBreakpoints();` inside `this._onEnablementOrModelChanged();` (once we have a model)

Reply:
`this._register(this._debugService.getModel().onDidChangeBreakpoints(() => {` is also inside `this._onEnablementOrModelChanged`